### ISSUE TITLE
Domains ending with a dot aren't recognized properly

### ISSUE THREAD

**Issue by laoghaire** [SecUSo/TORPEDO-Webextension#7]
![grafik](https://user-images.githubusercontent.com/26720249/109882208-d0267380-7c79-11eb-9a49-ef23b55d4bb7.png)

https://github.com/SecUSo/TORPEDO-Webextension/blob/f286bf7c2fd3c4b1087417b597039cb854ee347d/js/publicsuffixlist.js#L50
maybe simply add
``` javascript
if (hostname.charAt(hostname.length - 1) === ".") {
    hostname = hostname.slice(0, -1);
}
```

**Comment by philheller** [comment]
This issue was fixed with this pull request #8 which normalizes user input of the domain.
Thanks for notification @laoghaire .

**Comment by philheller** [comment]
Solved with the pull request #9.
Thank you for your contribution @laoghaire.
